opal/mca/pmix/pmix1xx/pmix/include/private/autogen/config.h.in

```diff
@@ -0,0 +1,636 @@
+/* include/private/autogen/config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
Yeah, I was on a webex last night with @dsolt showing him the Ways Of Git And Github, and we were running out of time -- so for expediency we did a `git add` of a whole tree.  I should have known better; oh well.  But there's still other things to fix in this PR, too; @dsolt and I will have a look on Monday.